ledger-api/grpc-definitions/com/daml/ledger/api/v1/command_service.proto

```diff
@@ -49,7 +49,7 @@ service CommandService {
   // or if an interpretation error occurred
   // - ``UNAVAILABLE``: if the participant is not yet ready to submit commands or if the service has been shut down.
   // - ``DEADLINE_EXCEEDED``: if the request failed to receive its completion within the predefined timeout.
-  rpc SubmitAndWaitForTransactionId (SubmitAndWaitRequest) returns (SubmitAndWaitForTransactionIdResponse);
+  rpc SubmitAndWaitForTransactionId (SubmitAndWaitRequest) returns (SubmitAndWaitForUpdateIdResponse);
```

Comment:
Do you want to rename the endpoint as well?
```suggestion
  rpc SubmitAndWaitForUpdateId (SubmitAndWaitRequest) returns (SubmitAndWaitForUpdateIdResponse);
```

Reply:
Not renaming here was a deliberate choice from my side: keeping the command_service for daml commands, and not caring here about transfers.
Of course this part needs much more love if we want to have this extended to transfers as well.

Reply:
I will rename this, to communicate the potential need for sync transfers.
For the PoC API first iteration we do not need this endpoint.